azure-pipelines.yml

```diff
@@ -147,6 +147,12 @@ jobs:
       ninja -C "$(sw_obj_dir sim-verilator)" all
       ninja -C "$(sw_obj_dir fpga)" all
     displayName: 'Build embedded targets'
+  - bash: |
+      # NOTE: Since tests are independent of the underlying target,
+      # we execute only one target for now. This will go away once
+      # we merge #1433.
+      ninja -C "$(sw_obj_dir fpga)" test
```

Comment:
I think you need a `. util/build_consts.sh` before this in this step.

Reply:
Oops.